CORE-351 update test data, activate and update e2e

bit-shube opened the pull request.

bit-shube committed:
CORE-351 update test data, activate and update e2e

jekutzsche reviewed (changes requested):
It is difficult to evaluate changes without an indication of why they were made or what they are intended to achieve. Since this also breaks a test, clarification is all the more necessary.

Something seems to be wrong with the line indentation.

bit-shube committed:
CORE-351 fix formatting issue, adapt IT to test data changes

bit-shube committed:
CORE-351 adapt IT

bit-shube committed:
CORE-351 activate e2es, adapt to new test data

bit-shube commented:
> It is difficult to evaluate changes without an indication of why they were made or what they are intended to achieve. Since this also breaks a test, clarification is all the more necessary.
> 
> Something seems to be wrong with the line indentation.

Right, finally go it to work: I modified the test data to make sure I always have an index case with 3 days of diary entries. I had to adapt the backend tests and the e2es.

I also fixed the formatting issue.

bit-shube committed:
CORE-351 fix formatting issue

jekutzsche reviewed (approved):
For me, the changes are fine, especially where now also the tests in the backend are fixed comprehensibly.

However, I would like to point out that possibly test cases of the testers for the manual tests could be based on the existing data.

bit-shube committed:
CORE-351 restore dayjs dependency

bit-shube committed:
CORE-351 fix test data, adapt test
